config/router-chatbot.js

```diff
@@ -132,12 +148,18 @@ router.post('/', (req, res) => {
       controller.debug(req, `sendMessage:${msg}`);
       controller.setCurrentCampaign(req.user, req.campaign_id);
 
-      mobilecommons.chatbot(req.body, mobilecommonsOip, msg);
+      let oip = process.env.MOBILECOMMONS_OIP_CAMPAIGNBOT;
+      if (controller.isCommand(req, 'member_support')) {
+        oip = process.env.MOBILECOMMONS_OIP_AGENTVIEW;
+      }
+
+      mobilecommons.chatbot(req.body, oip, msg);
 
       return res.send({ message: msg });
     })
     .catch(err => {
       controller.error(req, res, err);
+
```

Comment:
Oh, good. I missed (forgot?) this rule. Thank you for pointing it out, will be doing the same from now on!